src/Migrations/Version20180405030642.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types = 1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180405030642 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('ALTER TABLE comments ADD report_count BIGINT DEFAULT 0 NOT NULL');
+        $this->addSql('ALTER TABLE submissions ADD report_count BIGINT DEFAULT 0 NOT NULL');
+    }
+
+    public function down(Schema $schema)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('CREATE SCHEMA public');
```

Comment:
oh wow, doctrine:migrations:diff created that?
delete that line - we should use existing schema all the time - it's part of DBA, not code/dev

Reply:
Yeah I'll get rid of that. migrations:diff also adds a ton of "alter tables" that don't actually do anything. You can see them in some of the other more recent migrations. 

Version20180401054730 specifically

I've been manually removing those lines from the migrations.